src/brushes/pencil_brush.class.js

```diff
@@ -88,6 +110,7 @@
       if (!this.canvas._isMainEvent(options.e)) {
         return true;
       }
+      this.drawStraightLine = options.e[this.straightLineKey];
```

Comment:
wasn't this false by definition?

Reply:
can you rephrase?

Reply:
Do mean this?
https://github.com/fabricjs/fabric.js/pull/7034/files/54f4272aca08187618ade6ed8539cf763678378d#diff-62b861bab6725568506ad50661f0be7c7d3d79b8ba69229f664226fb54d9269bR23

Reply:
Weren't you setting this as false on mouse up, always? is not important for me, i just want to understand

Reply:
It is good practice I guess. 
16fd4e5